integration-tests/oidc-wiremock/src/main/java/io/quarkus/it/keycloak/CodeFlowUserInfoResource.java

```diff
@@ -33,7 +40,19 @@ public String access() {
 
     @GET
     @Path("/code-flow-user-info-github")
-    public String accessGitHub() {
+    public String accessGitHub(@QueryParam("propagate-bearer-token") Boolean propagateToServiceEndpoint) {
```

Comment:
Sorry Michal, what I really meant was that using these extra rest client calls in a wiremock test seems a bit excessive, can you just echo the injected access token (you can grab it from `@Inject AccessTokenCredential creds;`) and compare it against the value produced by the mock, add this value to what `access()` returns

Reply:
I am afraid I can't comprehend what you meant, but I provided 3 alternative tests that should be sufficient. Rest Client dependency is gone.